packages/components/addon/components/hds/pagination-bar/index.js

```diff
@@ -1,14 +1,22 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 export const TYPES = ['numbered', 'compact'];
 
 export default class HdsPaginationBarIndexComponent extends Component {
-  // UNCOMMENT THIS IF YOU NEED A CONSTRUCTOR
-  // constructor() {
-  //   super(...arguments);
-  //   // ADD YOUR ASSERTIONS HERE
-  // }
+  totalPages = Math.max(
+    Math.ceil(this.args.totalItems / this.args.itemsPerPage),
+    1
+  );
+  @tracked currentPage = 1;
```

Comment:
probably we may want to expose the possibility for the consumers to start from a specific page? (for example they store the current page in an URL, and the user reloads the page when `currentPage` is `3`)